cmd/manager/main.go

```diff
@@ -1,328 +1,31 @@
 package main
 
 import (
-	"context"
-	"flag"
-	"fmt"
-	"os"
-	"runtime"
-	"strings"
-
-	configclient "github.com/openshift/client-go/config/clientset/versioned"
-	operatorv1 "github.com/openshift/client-go/operator/clientset/versioned/typed/operator/v1"
-	"github.com/openshift/windows-machine-config-operator/pkg/apis"
-	"github.com/openshift/windows-machine-config-operator/pkg/clusternetwork"
-	"github.com/openshift/windows-machine-config-operator/pkg/controller"
-	wkl "github.com/openshift/windows-machine-config-operator/pkg/controller/wellknownlocations"
-	"github.com/openshift/windows-machine-config-operator/version"
-	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
-	kubemetrics "github.com/operator-framework/operator-sdk/pkg/kube-metrics"
-	"github.com/operator-framework/operator-sdk/pkg/leader"
-	"github.com/operator-framework/operator-sdk/pkg/log/zap"
-	"github.com/operator-framework/operator-sdk/pkg/metrics"
-	sdkVersion "github.com/operator-framework/operator-sdk/version"
-	"github.com/pkg/errors"
-	"github.com/spf13/pflag"
-	v1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/util/intstr"
-	"sigs.k8s.io/controller-runtime/pkg/cache"
-	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
-	_ "k8s.io/client-go/plugin/pkg/client/auth"
-	"k8s.io/client-go/rest"
-	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	"github.com/spf13/cobra"
```

Comment:
Do we really need to use cobra? Won't the simpler internal `flag` library suffice for what we are doing? Please see [flag.FlagSet](https://golang.org/pkg/flag/#FlagSet).

The amount of code churn you are introducing is also not desirable. We really don't need a `run` sub-command and even if we did, leaving things in `main.go` is fine for now. In general large code movements should be done in separate commits.

Reply:
reverted to use flagset instead of cobra